swabbie-redis/src/main/kotlin/com/netflix/spinnaker/swabbie/redis/RedisResourceTrackingRepository.kt

```diff
@@ -197,12 +190,7 @@ class RedisResourceTrackingRepository(
     try {
       markedResource = objectMapper.readValue(resource)
     } catch (e: Exception) {
-      registry.counter(
-        redisErrors.withTags(
-          "exception", e.javaClass.simpleName,
-          "action", "readMarkedResource"
-        )
-      ).increment()
+      registry.counter(registry.createId("redis.resourceTracking.errors")).increment()
```

Comment:
Worth pulling the creation of this ID up to the top of the class like we do elsewhere, so we don't "create" it twice?

Reply:
Late night PRs... saw that this morning.